src/Tribe/Attendees_Table.php

```diff
@@ -140,30 +109,30 @@ public function column_cb( $item ) {
 	}
 
 	/**
-	 * Handler for the order id column
+	 * Populates the purchaser column.
 	 *
-	 * @param $item
+	 * @param array $item
 	 *
 	 * @return string
 	 */
-	public function column_order_id( $item ) {
-
-		//back compat
-		if ( empty( $item['order_id_link'] ) ) {
-			$item['order_id_link'] = sprintf( '<a class="row-title" href="%s">%s</a>', esc_url( get_edit_post_link( $item['order_id'], true ) ), esc_html( $item['order_id'] ) );
-		}
-
-		return $item['order_id_link'];
+	public function column_purchaser( array $item ) {
+		$purchaser_name  = empty( $item[ 'purchaser_name' ] ) ? '' : esc_html( $item[ 'purchaser_name' ] );
+		$purchaser_email = empty( $item[ 'purchaser_email' ] ) ? '' : esc_html( $item[ 'purchaser_email' ] );
+
+		return "
+			<div class='purchaser_name'>{$purchaser_name}</div>
```

Comment:
Maybe it is github being weird, but I see a big gap between `{` and `$` on this line like so:

![screen shot 2016-07-26 at 4 59 50 pm](https://cloud.githubusercontent.com/assets/430385/17155279/6426d10c-5352-11e6-820f-3ab1ae0276d5.png)